feat: handle values passed via envvars

tecoholic opened the pull request:
### Description

Click [allows setting envvars](https://click.palletsprojects.com/en/stable/advanced/#detecting-the-source-of-a-parameter) to be used as a way to pass values to arguments and options. In such scenarios, the webui doesn't use the values from the environment it's run from.

In this PR, the default values in the forms are parsed and then overridden using the values from environment variables.

tecoholic committed:
feat: handle values passed via envvars

tecoholic commented:
@fredrik-corneliusson Hi, thank you for this amazing library. I thought I could add an improvement there. Can you kindly give this a review?

fredrik-corneliusson commented:
Thanks @tecoholic
I ran the tests and some failed. I have not investigated why. Please take a look and make sure all tests run by the check.sh script works. For more information about how to run in see the first section in:
https://github.com/fredrik-corneliusson/click-web/blob/master/CONTRIBUTING.rst

tecoholic commented:
@fredrik-corneliusson Thank you for responding. I apologize for missing the contributing guidelines. I will test everything locally and update the PR.

tecoholic committed:
test: add tests for the changes

tecoholic commented on check.sh:
```diff
@@ -1,7 +1,7 @@
 # Util command to run tests and code lint.
 
-pytest && isort . && flake8
+pytest && isort click_web tests && flake8 click_web tests
```
The `check.sh` kept messing with the virutalenv files at `.venv`. So I made it more explicit.

tecoholic commented:
@fredrik-corneliusson I updated the changes to make sure tests aren't failing and added a couple of tests for the newly introduced changes as well. Kindly take another look when you have some time.

tecoholic committed:
test: adds an extra test case handling multiple envvars
